examples/build.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
```

Comment:
Right, this is just a quick way to generate random data that can be
visualized in Emperor, good for simple tests. I've added some
documentation, so please let me know if you would like me to expand on
something.

On (Aug-19-15|23:02), mortonjt wrote:
> > @@ -0,0 +1,34 @@
> > +#!/usr/bin/env python
> 
> Think we can add some documentation to specify exactly what this build file is?  This is just a template right?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/407/files#r37497527